Remove os system cmd

spborder opened the pull request:
Fixes spaces/odd characters in file names issue. Creates importable "run_it" function. Updates names of some arguments for consistency.

sayatmimar commented on multic/cli/MultiCompartmentSegment/MultiCompartmentSegment.py:
```diff
@@ -1,23 +1,102 @@
+        'bordercrop':300,
+        'LR_region_pad':50,
+        'show_interstitium':True
+    }
```
We can move this outside of the function to the top. You can also call it `DEFAULT_VALS`. As default `args` are now stored in this dictionary, we don't need to parse them in `segmentation_school` right? If so let's delete parsing in `segmentation_school`.

sayatmimar reviewed (changes requested):
See my comment. Have you built the docker and tested this code in DSA?

spborder commented:
Fixed, updated image, tested on EC2 and all looks good.

Completed Job: http://ec2-3-230-122-132.compute-1.amazonaws.com:8080/#job/65de6cd8adb89a58fea10d7c
Test Image: http://ec2-3-230-122-132.compute-1.amazonaws.com:8080/#item/659c269b4e590bbcdeb406cc

sayatmimar reviewed (approved):
LGTM
